crates/compiler/parse/src/ident.rs

```diff
@@ -45,6 +45,8 @@ pub enum Ident<'a> {
     },
     /// `.foo { foo: 42 }` or `.1 (1, 2, 3)`
     AccessorFunction(Accessor<'a>),
+    /// `&foo { foo: 42 }, 3`
+    UpdaterFunction(&'a str),
```

Comment:
```suggestion
    /// `&foo { foo: 42 } 3`
    UpdaterFunction(&'a str),
```

Roc doesn't comma-delimit function applications